addons/crm/static/src/js/systray_activity_menu.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('crm.systray.ActivityMenu', function (require) {
+"use strict";
+
+var ActivityMenu = require('mail.systray.ActivityMenu');
+
+ActivityMenu.include({
+    /**
+     * @override
+     */
+    _getViewsList(model) {
+        if (model === "crm.lead") {
+            return this._super.apply(this, arguments).concat([[false, 'calendar'], [false, 'pivot'], [false, 'graph'], [false, 'activity']]);
```

Comment:
Line too long (ideally 80 soft limit, 100 hard limit), and here it's pretty easy to respect even 80 by moving the param to new line, and/or split the array in multi line.